3 guidelines java

hubertborysowski opened the pull request.

hubertborysowski committed:
Guidelines file commit.

hubertborysowski committed:
Update guidelines.md

hubertborysowski committed:
guidelines.md

Updated version of guidelines according to Mateusz's comments.

Mateusz-Dobrzynski commented on guidelines.md:
```diff
@@ -0,0 +1,62 @@
+## Programming Style
+
+For programming in Java we use [google style](https://google.github.io/styleguide/javaguide.html) 
```
Missing capital letter in `Google`

Mateusz-Dobrzynski commented on guidelines.md:
Put this file in the `doc` directory and change the name to uppercase, i.e. `CONTRIBUTING.md`, as in the project structure guide.

Mateusz-Dobrzynski commented on guidelines.md:
```diff
@@ -0,0 +1,62 @@
+
+For programming in Java we use [google style](https://google.github.io/styleguide/javaguide.html) 
+
+## Javadoc
```
This section is possibly redundant, as Google Style Guide includes a Javadoc section

hubertborysowski committed:
Delete guidelines.md

hubertborysowski committed:
3 Guidelines upload

hubertborysowski committed:
GUIDELINES path change

Changed path from 'meals-management-system/GUIDELINES.md' to 'meals-management-system/doc/GUIDELINES.md'.

hubertborysowski commented:
GUIDELINES.md path has just been changed. All the commented issues are resolved now.

hubertborysowski committed:
3 GUIDELINES.md path change; fixes

Mateusz-Dobrzynski reviewed (approved):
Change the file name to `CONTRIBUTING.md`
